index.rst

```diff
@@ -1,61 +1,546 @@
+only a single recommended setting. For example, for the ATDome CSC we have:
+
+::
+
+  recommendedSettingsLabels: test
+  recommendedSettingsVersion: v0.3.0-0-g6fbe3c7
+  settingsUrl: file:///home/saluser/repos/ts_config_attcs/ATDome/v1
+
+Some CSCs may also have multiple recommended settings, one of them being the
+preferred or default and another being secondary and so on.  In this case, the
+purpose of those configurations should be spelled out. As an example assume
+the ATAOS where we have a couple of available options for look-up tables. In
+this case, we may have something like:
+
+::
+
+  recommendedSettingsLabels: current,constant_hex,high_degree
+  recommendedSettingsVersion: v0.3.0-0-g6fbe3c7
+  settingsUrl: file:///home/saluser/repos/ts_config_attcs/ATAOS/v2
+
+Note how both ``recommendedSettingsVersion`` have the same value and they are,
+in fact, from the same configuration repo; ``ts_config_attcs``.
+
+Imagine now that during a test run, someone connects to the computer running
+the ATAOS CSC and edits the configuration. The ``recommendedSettingsVersion``
+would reflect that change with something like:
+
+::
+
+  recommendedSettingsVersion: v0.3.0-0-g6fbe3c7-dirty
+
+Even thought it may be useful to edit configurations on the fly for testing,
+the process should be avoided as much as possible. When this happen, it
+prevents us from precisely identifying what configuration was used.
+Alternatively, the user could create a branch on their work machine,
+make the required changes, commit, push it to github and pull/check out the new
+configuration in the CSC machine.
+
+For a CSC that does not uses a git repository for managing configuration
+version, and uses a database instead (like the ``ATCamera``), we may have
+something like:
+
+::
+
+  recommendedSettingsLabels: default,highgain_fast,lowgain_fast,highgain_slow,lowgain_slow
+  recommendedSettingsVersion: v10
+  settingsUrl:  sqlite:///home/camuser/config/config.db
+
+It might be the case where the configuration is hosted in a sql database which
+enables remote connection. Is this case, we could have something like:
+
+::
+
+  settingsUrl: mysql://10.0.100.104:3306/CONFIG
+
+Deriving new configuration parameters
+=====================================
+
+The process to derive new configuration parameters will vary considerably
+from component to component. In some cases, the configuration is simple enough
+that a change may involve simply replacing an IP or hostname value, a routine
+filter swap on an instrument or updating the limits to an axis range due to
+some evolving condition. On the other hand, in other cases, deriving new
+parameters may involve generating complex LUT that may require on sky
+observations and detailed data analysis.
+
+Following is a detail of each step of the process to update the CSC
+configuration.
+
+
+    #.  Create a Jira ticket to track the work being done. If details or
```

Comment:
Not sure if we can fix the formatting here, but it is driving me nuts that there is no whitespace above or below the numbered paragraphs, but there is for the paragraphs that are not numbered.   Don't suppose we can fix this somehow?

Reply:
I don't think it is possible to change that... At least I couldn't find any settings. But at the same time, I think this helps highlight the items in the numbered list.